web/src/utils/columns.ts

```diff
@@ -86,7 +86,7 @@ export const getDefaultColumns = (t: TFunction): Column[] => {
       name: t('Pod'),
       fieldName: 'SrcPod',
       isSelected: true,
-      filterType: FilterType.TEXT,
+      filterType: FilterType.K8S_LABEL,
```

Comment:
maybe i haven't understood exactly this change, but I find it weird to use "LABEL" here as what we are filtering is the pod name, not some label .. or do I miss something?

Reply:
you are right; that makes confusion with Loki labels :disappointed: 

The idea is to filter on any kubernetes id / name / label compliant with [Object Names and IDs](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names)
 
Would you prefer `K8S_NAMES` ?

Reply:
yeah I think it is less confusing with `K8S_NAMES`. The naming rule for "Kind" is probably even more strict than that, but I cannot find any spec on that